lib/template.js

```diff
@@ -118,9 +118,9 @@ class Template {
    * @return {Promise} - Promise for Template instance.
    */
   update () {
-    // Confirm template is a git repository. If not, throw an error.
+    // Confirm template is a git repository. If not, don't type to update.
     if (!fs.existsSync(path.join(this.path, '.git'))) {
-      return W.reject(new Error(`${this.name} is not a git repository`))
+      return W.resolve(this)
```

Comment:
If we did the commands that ping git for an update below would fail if it wasn't a git repo. This is basically saying if it's not a git repo move along, if it is try to update.